### DISCUSSION TITLE
Multiple cursor support `ge` operator

### DISCUSSION THREAD

**Discussion by tmacychen** [helix-editor/helix#9692] [General] [answered]
When using multi-selection, sometimes I want to  select the first one column or first two columns to the end of file.

I try to use `C` and `ge`, but not works.

Helix has implement the `gl`operator to the end of line  in `C` mode. So will it consider to implement `ge` ?

**Comment by pascalkuthe** [comment]
ge puts the cursor at the end of the document which naturally collapses them so it's working as intended and I dont see how that would make sense with multi cursors. Also multi cursors aren't a mode

**Reply by tmacychen** [reply]
I understand the `C`is not a mode but it  just looks like a mode.

I just want to select the first one column or first two columns to the end of file. 

In vim, I can use Ctrl + v into view mode and push button G to the end of line , and then use `j` or `k` control selected block .How can I do in Helix  editor ?

**Reply by pascalkuthe** [reply]
`ge` first and the repeat `A-C` as many times as you need to (or use a count)

**Reply by tmacychen** [reply]
That's why I want to open this issue. It looks like too inflexible. Right now, the acceptable way  is that lookup the  total number of lines  and type the number and push `C` button.

**Reply by pascalkuthe** [reply]
you need to articulate better what usecase you are trying to solve. I suspect you are just used to vims block selection mode and are trying to use helix multicursor in similar way but they are *not equivalent* and not meant to be.

**Reply by tmacychen** [reply]
Maybe helix can enhance the multi-selector,although it has already powerful.

**Reply by tmacychen** [reply]
Or Maybe it's  selection mode problem , it need to support block selection?

**Comment by the-mikedavis** [comment]
You can use `ge` in select mode (`v`) to select from the cursor to the last line of the document

**Reply by tmacychen** [reply]
Thanks for your reply, but i want to select the first one column or first two columns to the end of line in file. In vim, I can use Ctrl + v into view mode and push button G to the end of line , and then use j or k control selected block .How can I do in Helix editor ?

**Comment by tmacychen** [comment, marked as answer]
Finally , I find the helix-way to solve this issue. Use `%` select the entire file and push `K`,use `^` in regex. it can select the start of every lines.
